packages/common/src/ui/components/buttons/IconButton.tsx

```diff
@@ -1,36 +1,33 @@
 import React from 'react';
-import { Button, Tooltip } from '@mui/material';
-import { styled } from '@mui/material/styles';
+import { IconButton as MuiIconButton, Tooltip } from '@mui/material';
 import { LocaleKey, useTranslation } from '../../../intl/intlHelpers';
-import { DefaultButtonStyles } from './styles';
 
 interface ButtonProps {
   disabled?: boolean;
   icon: React.ReactNode;
   onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
-  size?: 'small' | 'medium';
-  titleKey: LocaleKey;
+  labelKey: LocaleKey;
 }
 
-const StyledButton = styled(Button)(({ theme }) => ({
-  ...DefaultButtonStyles,
-  boxShadow: theme.shadows[1],
-  color: theme.palette.primary.main,
-}));
-
-export const IconButton: React.FC<ButtonProps> = props => {
+export const IconButton: React.FC<ButtonProps> = ({
+  disabled,
+  icon,
+  onClick,
+  labelKey,
+}) => {
   const t = useTranslation();
-  const { disabled, icon, onClick, size, titleKey } = props;
+
+  const tooltip = t(labelKey);
```

Comment:
It seemed like half the buttons were using `labelKey` and half `titleKey` (Although it might have actually been more in favour of the one I didn't choose) so I tried to consolidate on a single one